src/ios/AKPlugin.m

```diff
@@ -60,8 +60,19 @@ - (void)loginWithEmail:(CDVInvokedUrlCommand *)command {
 }
 
 - (void)getAccount:(CDVInvokedUrlCommand *)command {
+  if (_accountKit == nil) {
+      _accountKit = [[AKFAccountKit alloc] initWithResponseType:AKFResponseTypeAccessToken];
```

Comment:
Shouldn't we return an error in such case? `getAccount` should always be called after any of the login methods imho (which initialize `_accountKit`)